Library/Homebrew/dev-cmd/bump-cask-pr.rb

```diff
@@ -98,85 +123,108 @@ def bump_cask_pr
       end
     end
 
-    if new_version.nil? && new_base_url.nil? && new_hash.nil?
+    new_version = if new_version.intel && new_version.arm
```

Comment:
If you wanted to keep `NewVersion` as a class: some of this logic could be moved there?

Reply:
Moved to NewVersion